bool visible
bool vertical # true implies vertical, false implies horizontal
geometry_msgs/Vector3 position
